media/css/externalpages/pocket/components/_mobile-nav.scss

```diff
@@ -0,0 +1,249 @@
+@import '../utils/variables';
+
+$icon-path: '/media/img/externalpages/pocket/mobile-nav/';
+
+// variables from pocket
+$easing-accelerate: cubic-bezier(0.4, 0, 1, 1);
+$easing-decelerate: cubic-bezier(0, 0, 0.2, 1);
+
+// this class will be added to the body to disable scroll while the nav is open
+.mobile-nav-open {
+    overflow: hidden;
+}
+
+.mobile-nav-wrapper {
+    background: rgba(26, 26, 26, 0.4) none repeat scroll 0 0;
+    bottom: 0;
+    display: none;
+    left: 0;
+    mix-blend-mode: normal;
+    opacity: 0;
+    pointer-events: none;
+    position: fixed;
+    right: 0;
+    transition: opacity 150ms $easing-accelerate, transform 75ms $easing-accelerate;
+    top: 0;
+    z-index: 20;
+
+    &.active {
+        display: block;
+        pointer-events: auto;
+    }
+}
+
+.mobile-nav {
+    background: $color-white;
+    border-radius: 0;
+    bottom: 0;
+    box-shadow: 0 4px 12px rgba(0, 0, 0, 0.12);
+    left: 0;
+    max-height: 100%;
+    max-width: 256px;
+    overflow-y: scroll;
+    position: absolute;
+    right: auto;
+    top: 0;
+    transform: translateX(-256px);
+    transition: all 250ms $easing-decelerate;
+    width: 100%;
+
+    &.active {
+        opacity: 1;
+        transform: translateX(0);
+        transition: all 250ms $easing-decelerate;
+    }
+
+    .mobile-nav-close {
+        align-items: center;
+        box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+        border-bottom: 1px solid #d9d9d9;
+        display: flex;
+        height: 64px;
+        margin-bottom: 8px;
+        padding: 0;
+
+        &:focus,
+        &:focus-within {
+            border: 1px solid $color-action-primary;
+        }
+
+        .mobile-nav-close-btn {
+            @include image-replaced;
+            background: url('#{$icon-path}/back-btn.svg') no-repeat transparent;
+            border: none;
+            color: $color-text-primary;
+            height: $spacing-lg;
+            margin: 0 $spacing-sm;
+            outline: none;
+            width: $spacing-lg;
+        }
+    }
+}
+
+.mobile-nav-list {
+    background: $color-white;
+    display: flex;
+    flex-direction: column;
+    font-family: $font-sans;
+    font-size: 16px;
+    font-weight: 500;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+
+    .mobile-nav-list-item {
+        margin: 0;
+        padding: 0;
+        width: 100%;
+
+        .mobile-nav-list-link {
+            align-items: center;
+            color: $color-text-primary;
+            fill: currentColor;
+            line-height: 24px;
+            display: inline-flex;
+            padding: 12px 16px;
+            position: relative;
+            text-decoration: none;
+            transition: background-color 100ms ease-out;
+            width: 100%;
+
+            &::before {
+                background-repeat: no-repeat;
+                content: '';
+                height: $spacing-lg;
+                width: $spacing-xl;
+            }
+
+            &.home {
```

Comment:
Add space above the `&::before` pseudo class. The rest of them on the file need a space above too ✨